.pipelines/Common.yml

```diff
@@ -31,7 +31,7 @@ stages:
         inputs:
           command: build
           projects: $(BuildSolution)
-          arguments: '--configuration $(BuildConfiguration)'
+          arguments: '--configuration $(BuildConfiguration) /p:PublicRelease=${{ parameters.official }}'
```

Comment:
This replaces the regex in the version file. The official build isn't triggered by a tag so I can't use the regex anymore.

Reply:
And versions are preserved because the mirror has identical history.